addons/message_whats_new_v2.11/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "api_version": "0.1",
+  "id": "message_whats_new_v2.11",
+  "name": "What's new message",
+  "type": "message",
+  "conditions": {
+    "min_client_version": "2.11.0",
+    "max_client_version": "2.11.9"
+  },  
+  "message": {
+    "id": "message_whats_new_v2.11",
+    "title": "Welcome to Mozilla VPN 2.11",
+    "subtitle": "You’re using the latest version of Mozilla VPN. Take a look at the newest features and updates:",
+    "date": 1668020390,
```

Comment:
This date is `Wednesday, November 9, 2022 6:59:50 PM` 
Does doing dates this way make sense? I feel like this should be generated client side... obviously out of scope for this ticket

Reply:
Correct! Good for now. We can do better in 2.12